sqlpage/templates/lazy.handlebars

```diff
@@ -0,0 +1,13 @@
+<div class="{{default class "my-2"}}"{{~#if style}} style="{{style}}" {{/if~}}>
```

Comment:
In all other components, `class` adds new classes without replacing the existing ones

Reply:
i know and i hesitate, this div is a light wrapper and i think user should be able to reset it easily if needed.

same for `<div class="{{default class "card my-2"}}">`. i add this default because most of the component are card.

should we just give no default and handle it like the style param ?